### PR TITLE
RPC/blockchain: Elaborate on scantxoutset documentation

### PR DESCRIPTION
**Pull request by luke-jr**
Minor doc improvements

### CODE REVIEW

File: src/rpc/blockchain.cpp [hunk 1]
```diff
@@ -2330,6 +2324,12 @@ static RPCHelpMan scantxoutset()
                 }},
                 {RPCResult::Type::STR_AMOUNT, "total_amount", "The total amount of all found unspent outputs in " + CURRENCY_UNIT},
             }},
+            RPCResult{"when action=='abort'", RPCResult::Type::BOOL, "success", "True if scan will be aborted (not necessarily before this RPC call returns), or false if there is no scan to abort."},
+            RPCResult{"when action=='status' and a scan is currently in progress", RPCResult::Type::OBJ, "", "",
+            {
+                {RPCResult::Type::NUM, "progress", "Approximate percent complete."},
```
Comment (jonatack) [inline src/rpc/blockchain.cpp:2330]: (the other fields in this RPC don't end with a period)
```suggestion
                {RPCResult::Type::NUM, "progress", "Approximate percent complete"},
```

File: src/rpc/blockchain.cpp [hunk 2]
```diff
@@ -2330,6 +2324,12 @@ static RPCHelpMan scantxoutset()
                 }},
                 {RPCResult::Type::STR_AMOUNT, "total_amount", "The total amount of all found unspent outputs in " + CURRENCY_UNIT},
             }},
+            RPCResult{"when action=='abort'", RPCResult::Type::BOOL, "success", "True if scan will be aborted (not necessarily before this RPC call returns), or false if there is no scan to abort."},
```
Comment (jonatack) [inline src/rpc/blockchain.cpp:2327]: nit, call is redundant with the meaning of RPC
```suggestion
            RPCResult{"when action=='abort'", RPCResult::Type::BOOL, "success", "True if scan will be aborted (not necessarily before this RPC returns), or false if there is no scan to abort."},
```

**Review by jonatack — COMMENTED** [review]
ACK modulo suggestions. I think the first two commits can be squashed, could also touch up the following line (see the line just above it `throw JSONRPCError(RPC_INVALID_PARAMETER, "Scan already in progress, use action \"abort\" or \"status\"");`)

```diff
--- a/src/rpc/blockchain.cpp
+++ b/src/rpc/blockchain.cpp
@@ -2103,7 +2103,7 @@ static RPCHelpMan scantxoutset()
         }
 
         if (request.params.size() < 2) {
-            throw JSONRPCError(RPC_MISC_ERROR, "scanobjects argument is required for the start action");
+            throw JSONRPCError(RPC_MISC_ERROR, "The \"scanobjects\" argument is required for the \"start\" action");
         }
```

**Comment by luke-jr** [comment]
> ```diff
> -            throw JSONRPCError(RPC_MISC_ERROR, "scanobjects argument is required for the start action");
>+            throw JSONRPCError(RPC_MISC_ERROR, "The \"scanobjects\" argument is required for the \"start\" action");
> ```

I think the current error is better...

**Comment by achow101** [comment]
ACK 7862c4ac4e7afc954ef9e2c7ee8c556e1654a374

**Pull request merged**